cmake/CcacheVersion.cmake

```diff
@@ -30,7 +32,8 @@ elseif(EXISTS "${CMAKE_SOURCE_DIR}/.git")
   # Scenario 3.
   find_package(Git QUIET)
   if(NOT GIT_FOUND)
-    message(SEND_ERROR "Could not find git")
+    set(VERSION "unknown")
+    message(WARNING "Could not find git")
```

Comment:
How about adding `return()` after this line to not execute the following commands with an unset `${GIT_EXCUTABLE}`?

Reply:
Good point. While it seems to do no harm, I'm actually not sure what happens. Maybe it tries to execute `describe`.

Reply:
I've moved the rest to 'else' to avoid a hidden return in the middle of that section.